Makefile

```diff
@@ -473,6 +504,32 @@ test.conformance:
 test.samples:
 	@cd config/samples/ && find . -not -name "kustomization.*" -type f | sort | xargs -I{} bash -c "echo;echo {}; kubectl apply -f {} && kubectl delete -f {}" \;
 
+.PHONY: test.charts.golden
+test.charts.golden:
+	@ \
+		$(MAKE) _chartsnap CHART=kong-operator || \
+	(echo "$$GOLDEN_TEST_FAILURE_MSG" && exit 1)
+
+.PHONY: test.charts.golden.update
+test.charts.golden.update:
+	helm repo update kong
```

Comment:
Is this required? Does this chart depend other charts in `kong` (the [kong/charts](https://github.com/Kong/charts/) repo)?

Reply:
Good catch, 08a56d18e60eba18579f80194ff8467bc540a05b covers that.